### DISCUSSION TITLE
REST client to call AWS API Gateway endpoints

### DISCUSSION THREAD

**Discussion by q3769-patientpoint** [awspring/spring-cloud-aws#878] [General]
Hello,

Is there a feature/plan to support Spring REST client integration to call endpoints hosted on AWS API Gateway? 

In addition to the current AUTH support of accessKey, accessSecret, region, etc., this would need to support signing the REST request with AWS sig V4 specs.

Thank you.

**Comment by maciejwalkowiak** [comment]
Hi! Apologies for late reply. We did not plan it, but it seems like a good idea to add it. Would you like to contribute?
